src/frontend/src/test-e2e/util.ts

```diff
@@ -57,6 +57,9 @@ export async function runInBrowserCommon(
     },
     automationProtocol: "webdriver",
     path: "/wd/hub",
+    logLevel: "info",
+    // outputDir pipes all webdriver log output into ./wdio.log
```

Comment:
```suggestion
    // outputDir pipes all webdriver log output into ./wdio.log
    // stdout only contains errors on test failures
```

Reply:
oh no, I just missed that comment! Will add it to a future PR